tests/installation/installation_overview.pm

```diff
@@ -71,12 +71,12 @@ sub run {
         ensure_ssh_unblocked;
         # Check the systemd target, see poo#45020
         # We need to exclude some scenarios where it doesn't work well
-        return if (is_caasp || is_upgrade || is_hyperv);
+        return if (is_caasp || is_upgrade || is_hyperv || get_var(REMOTE_CONTROLLER));
         if (get_var('DESKTOP')) {
             my $target = check_var('DESKTOP', 'textmode') ? "multi-user" : "graphical";
             select_console 'install-shell';
             # The default.target is not yet linked, so we have to parse the logs.
-            script_run("grep 'target has been set' /var/log/YaST2/y2log |tail -1 |grep --color=auto \"$target\"") && record_info "Could no detect the systemd target. Expected was: $target";
+            script_run("grep 'target has been set' /var/log/YaST2/y2log |tail -1 |grep --color=auto \"$target\"") && record_info("Warning: no target", "Could not detect the systemd target. Expected was: $target ( see poo#49622).");
```

Comment:
You could use `if` here instead of `&&`, it'd be more readable.

Reply:
I wouldn't find it more readable.

Reply:
I was referring readable in the sense that it's a really long line, in general you should break it, it could be with a simple enter if you want to keep the `&&`, use a condition as there are two different instruction or use variables for both side of the `&&` which is not recommended neither to use a lot.
But not big deal, optional.